docs/content/en/docs/topics/restore-a-backup/amazon-s3/_index.md

```diff
@@ -0,0 +1,400 @@
+```
+
+Allow a few minutes for the pods to start and proceed to a Ready state; check the pod status periodically:
+
+```bash
+kubectl get pods
+```
+
+**Output:**
+
+```bash
+NAME                                                   READY   STATUS      RESTARTS   AGE
+demo-cass-operator-65cc657-fq6bc                       1/1     Running     0          7m20s
+demo-dc1-default-sts-0                                 3/3     Running     0          6m53s
+demo-dc1-stargate-bb47877d5-54sdt                      1/1     Running     0          7m20s
+demo-grafana-7f84d96d47-xd79s                          2/2     Running     0          7m20s
+demo-kube-prometheus-stack-operator-76b984f9f4-pp745   1/1     Running     0          7m20s
+demo-medusa-operator-6888946787-qwzsx                  1/1     Running     0          7m20s
+demo-reaper-k8ssandra-656f5b77cc-nqfzv                 1/1     Running     0          4m21s
+demo-reaper-k8ssandra-schema-88cpx                     0/1     Completed   0          4m31s
+demo-reaper-operator-5b8c4c66b8-8cf86                  1/1     Running     0          7m20s
+prometheus-demo-kube-prometheus-stack-prometheus-0     2/2     Running     1          7m17s
+```
+
+Backup and restore operations are enabled by default. The `bucketSecret` corresponds to the secret credentials.
+
+```bash
+kubectl get cassdc dc1 -o yaml
+```
+
+In the output, see the `podTemplateSpec` property; two containers were added for Medusa.  Here’s the entry for the gRPC backup service:
+
+`    name: medusa`
+
+Here’s the entry for the restore’s init container. K8ssandra looks for an environment variable to be set, which would indicate when to perform a restore operation.
+
+`    name: medusa-restore`
+
+After a few minutes, once the pods have started, check the status:
+
+```bash
+kubectl get cassdc dc1 -o yaml
+```
+
+**Output:**
+
+```bash
+.
+.
+.
+status:
+  cassandraOperatorProgress: Ready
+  conditions:
+  ...
+  - lastTransitionTime: "2021-02-05T20:45:46Z"
+    message: ""
+    reason: ""
+    status: "True"
+    type: Ready
+  ...
+```
+
+### Add test data and get credentials for Cassandra access
+
+Now let’s create some test data.  The [test_data.cql](test_data.cql) sample file in GitHub contains:
+
+```sql
+CREATE KEYSPACE medusa_test  WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+USE medusa_test;
+CREATE TABLE users (email text primary key, name text, state text);
+insert into users (email, name, state) values ('alice@example.com', 'Alice Smith', 'TX');
+insert into users (email, name, state) values ('bob@example.com', 'Bob Jones', 'VA');
+insert into users (email, name, state) values ('carol@example.com', 'Carol Jackson', 'CA');
+insert into users (email, name, state) values ('david@example.com', 'David Yang', 'NV');
+```
+
+Copy the cql file to the K8ssandra StatefulSet (sts) pod:
+
+```bash
+kubectl cp test_data.cql demo-dc1-default-sts-0:/tmp -c cassandra
+```
```

Comment:
We're trying to move away from suggesting to copy a cql file into the container, and rather copy/paste the queries directly to CQLSH. You can steal the recommendation [here](https://github.com/k8ssandra/k8ssandra/pull/605/files/1ef93edac3b26ba3337093df3a572faa1b20a54e#r606143427).

Reply:
Ok yeah, better. Will edit my version here. Thanks Alex.